packages/codemod/src/codemod.ts

```diff
@@ -85,9 +85,9 @@ export function babelRecast({
 
     return {
       // @ts-expect-error
-      warnings: metadata ? metadata.warnings : [],
+      warnings: metadata && metadata.warnings ? metadata.warnings : [],
```

Comment:
```suggestion
      warnings: metadata?.warnings ?? [],
```

Gold plating (🏅) is my passion